core/rint/src/TRint.cxx

```diff
@@ -181,7 +181,7 @@ TRint::TRint(const char *appClassName, Int_t *argc, char **argv, void *options,
          std::cerr << "root: unrecognized option '" << argv[n] << "'\n";
       }
       std::cerr << "Try 'root --help' for more information.\n";
-      TApplication::Terminate(0);
+      TApplication::Terminate(2);
```

Comment:
Just for curiosity: why 2? (and not, say, 1)

Reply:
Please see description at the top:
https://github.com/root-project/root/pull/19196#issue-3178852646

Reply:
Ah sorry, I missed that sentence.